core/src/main/scala/pureconfig/ConvertHelpers.scala

```diff
@@ -94,36 +102,68 @@ trait ConvertHelpers {
     override def to(t: T): ConfigValue = ConfigValueFactory.fromAnyRef(t)
   }
 
-  def fromStringReader[T](fromF: String => Option[ConfigValueLocation] => Either[ConfigReaderFailure, T]): ConfigConvert[T] = new ConfigConvert[T] {
+  def fromStringReader[T](fromF: String => Option[ConfigValueLocation] => Either[ConfigReaderFailure, T]): ConfigReader[T] = new ConfigReader[T] {
```

Comment:
Yes, that's a good idea! I'll move all the methods, but I'll leave forwarder deprecated methods in `ConfigConvert` in order not to break compatibility here, ok?